schema.graphql

```diff
@@ -2,6 +2,9 @@
 # NFTs being lent out on the Rent NFT platform"
 type Lending @entity {
   id: ID!
+  # Timestamp of the transaction. Use this for pagination, filtering
```

Comment:
I know this came from whoopi subgraph but `cursor` is not a timestamp. It is a counter. We should update the comment here accordingly. It can be re-used on the other subgraphs.